### ISSUE TITLE
Documentations update

### ISSUE THREAD

**Issue by ArchangeGabriel** [Bumblebee-Project/Bumblebee#772]
With 4.0 on track, they are several documentation updates needed around there.

I’ll try to list them here, feel free to add other items. For now I see:
- [ ] Update wikis about false/noise errors:
    - [ ] Github.com
    - [ ] ArchLinux
    - [ ] Ubuntu
    - [ ] Fedora?
- [ ] Update wikis about multi-monitor (especially in link with #720):
    - [ ] Github.com
    - [ ] ArchLinux
    - [ ] Ubuntu
    - [ ] Fedora?
- [ ] Add FAQ entries about Vulkan pointing to #769 
- [ ] Add FAQ entries about PRIME to explain the differences
- [ ] Add FAQ entries about PCIe bottlenecking/rewrite entries about performances
- [ ] Update instructions for packagers

Also, we could make use of newer GitHub functions:
- https://github.com/Bumblebee-Project/Bumblebee/wiki/Reporting-Issues could be (partially?) replaced by https://help.github.com/articles/creating-an-issue-template-for-your-repository/.
- https://github.com/Bumblebee-Project/Bumblebee/wiki/Contributing-and-joining-the-team could be similarly replaced by https://help.github.com/articles/setting-guidelines-for-repository-contributors/.

We may change http://bumblebee-project.org/ for bumblebee-project.github.io using https://help.github.com/categories/github-pages-basics/, so that @Thulinma could stop paying for the domain and having to host us.

**Comment by gsgatlin** [comment]
Here is the section I made about multi monitor on fedora:

https://fedoraproject.org/wiki/Bumblebee#Multi_monitor_setup_with_closed_driver.2C_fedora_22_or_later

I can change whatever in those docs. Just let me know what should be changed.

**Comment by Thulinma** [comment]
FYI: I'm not actually paying for the domain. I believe that's @Lekensteyn ?
The host is not an issue and can stay up for many more years without problem, but if you want to move it that's fine too of course.

**Comment by ArchangeGabriel** [comment]
Oh, OK. Yeah, I could host it too without problem, but it was more about the need to pay a domain name for nothing and also have a far easier way to handle site management. ;)

**Comment by Lekensteyn** [comment]
Yes, I am paying for the .org domain, it is no problem. There was a git repo on the server for the web content, maybe we should put it on Bumblebee-Project/bumblebee-web? A post-commit hook could then somehow ping the server and trigger a pull.

What about the tarballs? I did not check them in when creating the git repo, they are untracked by git, but must still be uploaded for releases.

@Thulinma What is the SSH username? I tried peter, lekensteyn and bumblebee, but the key was rejected.

**Comment by bluca** [comment]
What about using Github + Travis for the release tarballs? Free hosting and bandwidth, and doing a release is just the matter of pushing a git tag. If you guys want, I can look into it next week.

**Comment by Thulinma** [comment]
@Lekensteyn I actually think that server doesn't allow logins for the bumblebee data at all right now. I probably forgot to add the user when moving hosts last year. I'll take a look tonight and will check.

**Comment by ArchangeGabriel** [comment]
@bluca That’s a good idea.

@Lekensteyn We could just use Bumblebee-Project/Bumblebee-Project to get the automated bumblebee-project.github.io website feature like proposed above. Likewise, the hook is really trivial, and with bluca proposal the release tarballs should be an easy thing.

**Comment by ArchangeGabriel** [comment]
@gsgatlin I’m not sure what is the current status of multi-monitor in fact. The change I was referring to is the changes needed in xorg.conf.nvidia and 10-input-drivers.conf.

**Comment by danger89** [comment]
Can somebody upload the website on github of http://bumblebee-project.org/ maybe adding a FAQ?

**Comment by Thulinma** [comment]
@danger89 I would upload it all to a organization page, but it appears I don't have the permissions required to create the repo for this.
@ArchangeGabriel Could you either create a bumblebee-project.github.io repo or give me the access needed to do it myself? I'll then upload the webroot to it, so it becomes available at http://bumblebee-project.github.io which the domain can then be redirected to.

**Comment by ArchangeGabriel** [comment]
@Thulinma: You should have all rights now. ;)

**Comment by Thulinma** [comment]
@ArchangeGabriel Thanks!

Working now: https://bumblebee-project.github.io/
Source is in the Bumblebee-Project.github.io repo.

**Comment by Lekensteyn** [comment]
I'll change the CNAME, currently checking whether I can make this work with SSL.

**Comment by rafaelcn** [comment]
Just to be aware of things, does bumblebee team will pursue the development for 4.0?

**Comment by Lekensteyn** [comment]
@rafaelcn There will be a 4.0, but at the moment it does not seem to be a high priority.

**Comment by rafaelcn** [comment]
Thanks for the info. I want to work with this piece of Bumblebee software as well :-)

**Comment by danger89** [comment]
I see still quite a bit of views on the FB page of [Bumblebee](https://www.facebook.com/BumblebeeProject), while I do not post any update since there is nothing to tell.

But still we have visits almost each day... Meaning Bumblebee is still searched on 
![fb_views](https://cloud.githubusercontent.com/assets/628926/25810512/9b413818-3410-11e7-8e5a-9189ff947173.png)

Google Confirms this via Google Trends. 'Bumblebee Linux' -> **Blue** vs 'Nvidia Prime' -> **Red**.

![google_trends](https://cloud.githubusercontent.com/assets/628926/25810737/50691de6-3411-11e7-8f29-66b7749a38bd.png)
